packages/mobile/src/screens/SingleMovie2.tsx

```diff
@@ -0,0 +1,166 @@
+import back_arrow from '../../assets/img/back_arrow2.png';
+import h from '../../assets/img/h.png';
+import play from '../../assets/img/play_brown.png';
+import {
+  Container,
+  Player,
+  PlayerImage,
+  BackButton,
+  BackArrowImage,
+  ColumContainerLeft,
+  ColumContainerRight,
+  ImageList,
+  ImageItem,
+  MovieName,
+  MovieKind,
+  RowContainer,
+  Background,
+  MovieTime,
+  Owerview,
+  Description,
+  Heart,
+  HeartRatio,
+  RatioContainer,
+  Separator,
+  ViewImage,
+  StarsContainer,
+  Star,
+  VoteText,
+  PlayButton,
+  Triangle,
+} from './SingleMovieStyle';
+import AppNavigation from './../components/AppNavigation';
+
+const DATA = [
+  {
+    name: 'Joker',
+    kind: 'Drama-comedy',
+    runTime: '2h 2min',
+    ratio: '95%',
+    id: 'bd7acbea-c1b1-46c2-aed5-3ad53abb28ba',
+    uri: 'https://encrypted-tbn0.gstatic.com/images?q=tbn%3AANd9GcQsAIA_fyXZtUCCy2_ZtuKQKSYNf4mJhnxn5tXojNVvELTCpGr-',
+  },
+  {
+    name: 'Joker',
+    kind: 'Drama-comedy',
+    runTime: '2h 2min',
+    ratio: '95%',
+    id: '3ac68afc-c605-48d3-a4f8-fbd91aa97f63',
+    uri: 'https://i.pinimg.com/originals/df/67/7f/df677fed943171dc7c75276544753586.jpg',
+  },
+  {
+    name: 'Joker',
+    kind: 'Drama-comedy',
+    runTime: '2h 2min',
+    ratio: '95%',
+    id: '58694a0f-3da1-471f-bd96-145571e29d72',
+    uri: 'https://encrypted-tbn0.gstatic.com/images?q=tbn%3AANd9GcTGAwxu1pY8sMoyT95ZsXBQS1hx4vTT59jf_t2oFPK14_oir47R',
+  },
+  {
+    name: 'Joker',
+    kind: 'Drama-comedy',
+    runTime: '2h 2min',
+    ratio: '95%',
+    id: 'bd7acbea-c1b1-46c2-aed5-3ad53abb28b2',
+    uri: 'https://encrypted-tbn0.gstatic.com/images?q=tbn%3AANd9GcQsiLoYimfu2253MofOAYbbeFnJkcDOR4047PIld3tc5e3Q_ppL',
+  },
+  {
+    name: 'Joker',
+    kind: 'Drama-comedy',
+    runTime: '2h 2min',
+    ratio: '95%',
+    id: '3ac68afc-c605-48d3-a4f8-fbd91aa97f61',
+    uri: 'https://encrypted-tbn0.gstatic.com/images?q=tbn%3AANd9GcTGAwxu1pY8sMoyT95ZsXBQS1hx4vTT59jf_t2oFPK14_oir47R',
+  },
+  {
+    name: 'Joker',
+    kind: 'Drama-comedy',
+    runTime: '2h 2min',
+    ratio: '95%',
+    id: '58694a0f-3da1-471f-bd96-145571e29d71',
+    uri: 'https://encrypted-tbn0.gstatic.com/images?q=tbn%3AANd9GcTGAwxu1pY8sMoyT95ZsXBQS1hx4vTT59jf_t2oFPK14_oir47R',
+  },
+];
+
+export default class SingleMOvie2 extends React.Component {
+  static navigationOptions = ({navigation}) => {
+   return {
+    headerShown: false,
+  };
+  };
+
+  render() {
+    return (
+      <Background>
+        <Container>
+          <Player>
+            <PlayerImage source={player} />
+          </Player>
+
+          <ImageList>
+            <FlatList
+              snapToAlignment={'start'}
+              snapToInterval={359 + 10}
+              decelerationRate={'fast'}
+              ItemSeparatorComponent={() => <Separator />}
+              showsHorizontalScrollIndicator={false}
+              horizontal={true}
+              data={DATA}
+              renderItem={({item}) => (
```

Comment:
pozbądz się tych definicji funkcji strzałkowych z props. Jest to dozwolone tylko gdy nie masz jak inaczej przekazać tam jakiejś wartości np. id przycisku najczesciej towarzyszy temu map.

Reply:
Wydziel je jako metody/własciwości tego obiektu albo komponenty funkcyjne